packages/orchestrator-ui-components/src/components/WfoStartButton/WfoStartTaskComboBox.tsx

```diff
@@ -42,6 +42,7 @@ export const WfoStartTaskButtonComboBox = () => {
             options={comboBoxOptions}
             onOptionChange={handleOptionChange}
             isProcess={false}
+            width="600px"
```

Comment:
Consider using the `css` prop instead of a value. Therefore you need to add the className prop to WfoStartButtonComboBoxProps

Reply:
Thought you meant use **!important** in css there for a while. Wasn't it **!discuss** instead of **!important**

Reply:
Yeah I used the should-never-use `!importtant` to mark something important for discussion. Might not be a good idea in a css context discussion here. Sticking to discuss from now on.

On topic: try to forward the css prop by either: 
- forwarding the `css` prop itself which has the type `SerializedStyles` from `@emotion/react`
- forwarding the `className` prop which also enables the css prop to be used